docs/index.html

```diff
@@ -1,73 +1,101 @@
-    name: '<div style="font-size: 25px; font-style: italic;" id="navbar"><span style="font-weight: bold; color: var(--heading-color)" >booster</span> framework</div>',
-    maxLevel: 4,
-    themeColor: '#5A6FFF',
-    auto2top:true,
-    loadSidebar: 'sidebar.md',
-    loadNavbar: 'navbar.md',
-    mergeNavbar: true,
-    subMaxLevel: 4,
-    sidebarDisplayLevel:1,
-    search: {
-      noData: 'No results!',
-      paths: 'auto',
-      placeholder: 'Search',
-    },
-    copyCode: {
-      buttonText : 'Copy to clipboard',
-      errorText  : 'Error',
-      successText: 'Copied'
-    },
-    remoteMarkdown: {
-      tag: 'contributing',
-    },
-    plugins: [
-      function(hook, vm) {
-        hook.beforeEach(function (html) {
-          const file = vm.router.getFile().replace('/booster/','');
-          let url = 'https://github.com/boostercloud/booster/tree/main/docs/' + file;
-          if (file.includes('CONTRIBUTING')) {
-            url = 'https://github.com/boostercloud/booster/tree/main/CONTRIBUTING.md'
+    window.$docsify = {
+        homepage: 'chapters/01_introduction.md',
+        name: '<div style="font-size: 25px; font-style: italic;" id="navbar"><span style="font-weight: bold; color: var(--heading-color)" >booster</span> framework</div>',
+        maxLevel: 4,
+        themeColor: '#5A6FFF',
+        auto2top: true,
+        loadSidebar: 'sidebar.md',
+        loadNavbar: 'navbar.md',
+        mergeNavbar: true,
+        subMaxLevel: 4,
+        sidebarDisplayLevel: 1,
+        search: {
+            noData: 'No results!',
+            paths: 'auto',
+            placeholder: 'Search',
+        },
+        copyCode: {
+            buttonText: 'Copy to clipboard',
+            errorText: 'Error',
+            successText: 'Copied'
+        },
+        remoteMarkdown: {
+            tag: 'contributing',
+        },
+        plugins: [
+            function (hook, vm) {
+                hook.beforeEach(function (html) {
+                    const file = vm.router.getFile().replace('/booster/', '')
+                    let url = 'https://github.com/boostercloud/booster/tree/main/docs/' + file
+                    if (file.includes('CONTRIBUTING')) {
+                        url = 'https://github.com/boostercloud/booster/tree/main/CONTRIBUTING.md'
+                    }
+                    return '[📝Edit Document](' + url + ')\n' + html
+                })
+                const footer = [
+                    '<footer style="text-align: center;margin-top: 50px;">',
+                    '<span> Booster is a spin-off project from <a href="https://www.theagilemonkeys.com">The Agile Monkeys</a>.</span>',
+                    '</footer>'
+                ].join('')
+                hook.afterEach(function (html) {
+                    return html + footer
+                })
+            },
+          function meta(name, content) {
+            document.querySelector(`meta[name="${name}"]`).content = content || ''
```

Comment:
I believe this function is throwing the error, since the element searched with querySelector is not found. Do you know when this function is executed? Is the DOM loaded before this function is run?